pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -75,6 +77,24 @@ func makeSegmentstorePodSpec(p *api.PravegaCluster) corev1.PodSpec {
 			},
 		},
 	}
+	if len(configMapName) > 0 {
+		environment = append(environment, corev1.EnvFromSource{
+			ConfigMapRef: &corev1.ConfigMapEnvSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: configMapName,
+				},
+			},
+		})
+	}
+	if len(secretName) > 0 && !p.Spec.Pravega.SegmentStoreSecret.MountToVolume {
+		environment = append(environment, corev1.EnvFromSource{
+			SecretRef: &corev1.SecretEnvSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: secretName,
+				},
+			},
+		})
+	}
```

Comment:
Same comment as len(configMapName)

Reply:
Do you think this bool variable -`p.Spec.Pravega.SegmentStoreSecret.MountToVolume` is really necessary? If VolumeMountPath is not specified, but SecretName is specified, we could assume they want to expose as envvars... else use mountPath